src/modules/share/share.repository.ts

```diff
@@ -142,7 +158,15 @@ export class SequelizeShareRepository implements ShareRepository {
   ): Promise<Share | null> {
     const share = await this.shareModel.findOne({
       where: { fileId, userId },
-      include: [this.fileModel, this.userModel],
+      include: [
+        this.userModel,
+        {
+          model: this.fileModel,
+          where: {
+            deleted: false,
+          },
```

Comment:
Why do the others have required: false and this one doesn't ?

What does required do?